src/views/teacherregistration/teacherregistration.jsx

```diff
@@ -47,34 +51,35 @@ class TeacherRegistration extends React.Component {
             method: 'post',
             useCsrf: true,
             formData: {
-                username: this.state.formData.user.username,
-                email: formData.user.email,
-                password: this.state.formData.user.password,
-                birth_month: this.state.formData.user.birth.month,
-                birth_year: this.state.formData.user.birth.year,
-                gender: (
+                'username': this.state.formData.user.username,
+                'email': formData.user.email,
+                'g-recaptcha-response': formData['g-recaptcha-response'],
```

Comment:
g-captcha-response is the new line. The rest are just adding ' ' around the attributes because this one has a - in it and lint says to be consistent.